staff/sergi-pallice/course-1/Task1/contact.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+  <link rel="stylesheet" href="styles.css">
+<title>NYSL</title>
+</head>
+<body>
+  <div class="index-frame1">
+    <img src="Images/norcal_logo.png" class="logo"/>
+    <h1>NorCal Time Trial Series</h1>
+    <h3><a href="index.html">Home</a> | <a href="about.html">About </a>| <a href="contact.html">Contact</a></h3>
+  </div>
+<p>Please email us at info@nctts.org</p>
```

Comment:
This should go inside a div and image 3 is missing. Also email should be a link to the email.